go/libkb/leveldb_cleaner.go

```diff
@@ -273,13 +290,14 @@ func (c *levelDbCleaner) cleanBatch(startKey []byte) (int, []byte, error) {
 	batch := new(leveldb.Batch)
 	for batch.Len() < 1000 && iter.Next() {
 		key := iter.Key()
-		if _, found := c.cache.Get(c.cacheKey(key)); !found {
+		cache := c.getCache()
```

Comment:
does this block of code make sense? though it might not matter as any races would be during shutdown

Reply:
hmm if we shutdown this loop should get kill though